setup.py

```diff
@@ -20,6 +20,7 @@
 
 sqlite_requires = ['sqlalchemy>=1.1,<1.3.7']
 postgres_requires = sqlite_requires + ['psycopg2']
+mssql_requires = sqlite_requires + ['pyodbc']
```

Comment:
In general, I wonder if there is a way to do optional dependencies? i.e., an ibis user that doesn't use it with mssql shouldn't need to install `pyodbc`?

Reply:
This is exactly how this is already configured. To install mssql you would `pip install ibis[mssql]`